pkg/server/plugin/upstreamauthority/disk/disk_test.go

```diff
@@ -288,13 +300,21 @@ func (s *DiskSuite) TestSubmitInvalidCSR() {
 		csr, _, err := util.NewCSRTemplate(invalidSpiffeID)
 		require.NoError(err)
 
-		resp, err := s.p.MintX509CA(ctx, &upstreamauthority.MintX509CARequest{Csr: csr})
+		stream, err := s.p.MintX509CA(ctx, &upstreamauthority.MintX509CARequest{Csr: csr})
+		require.NoError(err)
+		require.NotNil(stream)
+
+		resp, err := stream.Recv()
 		require.Error(err)
 		require.Nil(resp)
 	}
 
 	invalidSequenceOfBytesAsCSR := []byte("invalid-csr")
-	resp, err := s.p.MintX509CA(ctx, &upstreamauthority.MintX509CARequest{Csr: invalidSequenceOfBytesAsCSR})
+	stream, err := s.p.MintX509CA(ctx, &upstreamauthority.MintX509CARequest{Csr: invalidSequenceOfBytesAsCSR})
+	require.NoError(err)
+	require.NotNil(stream)
+
+	resp, err := stream.Recv()
```

Comment:
We should add an additional call to Recv() somewhere after here that we assert returns io.EOF.